### PR TITLE
Yoshimura ho collab

### PR DESCRIPTION
**Pull request by xsongit12**
kappa

### CODE REVIEW

**Comment by thyo9470** [comment]
Resolve that shtuff!!!!

**Comment by Dyan654** [comment]
Looks ok to me, but you should add test cases in test.cpp

File: math/FunMath.cpp [hunk 1]
```diff
@@ -1 +1,21 @@
 #include "FunMath.h"
+
+/**
+    Adds all element from a that do not match with b
```
Comment (muzny) [inline math/FunMath.cpp]: This comment is confusing—are we removing elements from a that **are** in b or that **aren't** in b?

You are missing comments about the type and nature of parameters and return values.

File: math/FunMath.cpp [hunk 2]
```diff
@@ -1 +1,21 @@
...
+std::vector<int> MatchVectors(std::vector<int> a, std::vector<int> b){
+    std::vector<int> c;
+    bool match;
+    for(int i = 0; i < a.size(); i++){
+        match = false;
+        for(int j = 0; j < b.size(); j++){
+            if(a[i] == b[j]){
+                match = true;
+            }
+        }
```
Comment (muzny) [inline math/FunMath.cpp]: 1 -2 inline comments would be useful here to explain your algorithm

**Pull request merged**